src/compas_robots/viewer/scene/robotmodelobject.py

```diff
@@ -52,12 +53,18 @@ def __init__(
         self.hide_coplanaredges = hide_coplanaredges
         self._show_visual = show_visual or True
         self._show_collision = show_collision or False
-        self.configuration: Configuration = configuration or model.zero_configuration()
-        super(RobotModelObject, self).__init__(model=model, **kwargs)
+        self.configuration: Configuration = configuration or item.zero_configuration()
+        super(RobotModelObject, self).__init__(model=item, **kwargs)
```

Comment:
This is not right to me. The model should be passed in as `item` in kwargs.  There should be neither a positional or keyword argument. 
This line should simply be
```python
super(RobotModelObject, self).__init__(**kwargs)
```

Then in the init function of `BaseRobotModelObject`, https://github.com/PingHsunTsai/compas_robots/blob/4124bc62000ecc09dc3d9e21167fd020637ec6f5/src/compas_robots/scene/baserobotmodelobject.py#L77 
pls change it to 
```python
    def __init__(self,  **kwargs):
        super(BaseRobotModelObject, self).__init__(**kwargs)
        self.create()
        self.scale_factor = 1.0
        self.attached_tool_models = {}
        self.attached_items = {}
    
    @property
    def model(self):
        return self.item
```